test/e2e/config/operator.yaml

```diff
@@ -49,7 +49,7 @@ variables:
   TURTLES_PATH: "turtles/rancher-turtles"
   TURTLES_REPO_NAME: "turtles"
   TURTLES_URL: https://rancher.github.io/turtles
-  CPI_IMAGE_K8S_VERSION: "v1.30.0"
+  CPI_IMAGE_K8S_VERSION: "v1.30.4"
```

Comment:
I can't find this version of CPI image in gcr: https://console.cloud.google.com/gcr/images/cloud-provider-vsphere/GLOBAL/cpi/release/manager

Reply:
I reduced requirements to version 1.30.1 here, thanks.